packages/bentley/grouping-mapping-widget/src/widget/utils.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+type Falsy = false | 0 | "" | null | undefined;
```

Comment:
where are these being used? i know this was a aprt of the demo-portal but didnt see it being used anywhere

Reply:
Left over from demo portal will remove.